### ISSUE TITLE
Questions about default run settings

### ISSUE THREAD

**Issue by loftusa** [Janspiry/Image-Super-Resolution-via-Iterative-Refinement#60]
Hi, I have been running on the basic celebs dataset you guys include in this repo. I have a few related questions:

1. The ffhq_64_512 training dataset appears to only contain four images. Why are there so few images being trained on?
2. I completed a run in about 8 hours on these four images. Why did it take so long to train on them? Does the noising/denoising process really add that much extra time to training?
3. I got good results when testing on the validation set. How is this possible, given that there were only four images in the training set? I had assumed that the model would have severely overfit.

**Comment by Janspiry** [comment]
![image](https://user-images.githubusercontent.com/32922494/163514773-32bb1050-0415-4e77-b753-004a19ba2904.png)
1. Four images are just for formatting reference, and if you don't have your own training data, you can refer to the guidelines.
2. What do you mean a run? I think it will not take so long in a epoch.
3. I think so....

**Comment by loftusa** [comment]
> ![image](https://user-images.githubusercontent.com/32922494/163514773-32bb1050-0415-4e77-b753-004a19ba2904.png)
> 
> 1. Four images are just for formatting reference, and if you don't have your own training data, you can refer to the guidelines.
> 2. What do you mean a run? I think it will not take so long in a epoch.
> 3. I think so....

Thanks, didn't see that! Here are results from the validation set when I trained with just those four images. I was confused as to why the results were so good.
<img width="745" alt="Capture" src="https://user-images.githubusercontent.com/12386450/163692674-6c7b8d6b-26ba-4544-9402-144e13bb657f.PNG">

2. by "a run", I meant running `sr.py` with the command `%run sr.py -p train -c config/sr_sr4_64_512.json -enable_wandb -log_wandb_ckpt -log_eval` until the script stopped (I am assuming when the loss converged).

3. Meaning, you think it overfit?

**Comment by Janspiry** [comment]
I see,  and this is indeed a very interesting phenomenon, have you tested the other images of the validation set? Because of the random noise, I don't think the training will overfit, but the diversity of the generation will be limited.
